appengine/standard/firebase/firetactoe/templates/fire_index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+  <head>
+    {% include "_firebase_config.html" %}
+    <link rel="stylesheet" type="text/css" href="static/main.css" />
+    <script src="static/main.js"></script>
+    <script>
+      initGame('{{ game_key }}', '{{ me }}', '{{ token }}', '{{ channel_id }}',
```

Comment:
You can install invoke the function:
```
var $config = $('something),
    game_key = $config.data('game-key'),
    ...;

initGame(game_key, ...);
```

It just decouples the JS from the template engine - I really, really abhor mixing server-side template interpolation in javascript.